CHANGELOG.md

```diff
@@ -8,7 +8,7 @@
 
 ### Changed
 
-- nothing changed
+- changed serializer fields validations
```

Comment:
Description of changes is not specific enough

Reply:
added more details